src/dex-ui/pages/CreateProposal/CreateProposal.tsx

```diff
@@ -38,29 +24,12 @@ export const CreateProposal = (props: CreateProposalProps) => {
    * In case the user bookmarks the URL
    * */
   const { proposalType } = props;
-
```

Comment:
Form state is now managed within the Proposal Form components using `react-hook-form`.